src/utils/conversion.ts

```diff
@@ -1601,8 +1601,8 @@ export function stringToFundingRoundName(roundName: string, context: Context): F
 /**
  * @hidden
  */
-export function fundingRoundNameToString(roundName: FundingRoundName): string {
-  return roundName.toString();
+export function fundingRoundNameToString(roundName: FundingRoundName): string | null {
```

Comment:
I'd rather you perform the isEmpty check in `currentFoundingRound` and leave this as it was. It'll be easier to adapt when core changes the return type to `Option`